README.md

```diff
@@ -18,6 +18,25 @@ then create a new document
     doc = Govspeak::Document.new "^Test^"
     puts doc.to_html
 
+## Changes appearing across GOV.UK
+
+Some additional steps or considerations are needed to ensure changes to govspeak cascade across GOV.UK a holistic way.
+
+Once govspeak has been updated and version incremented then: 
+- [`govuk_publishing_components` govspeak](https://components.publishing.service.gov.uk/component-guide/govspeak) will also need updating
+- Publishing apps, including [content-publisher](https://github.com/alphagov/content-publisher) & [whitehall]( 
```

Comment:
It would be ideal if you could link to a list of publishing apps here, if one exists. I worry that listing just whitehall and content-pub will encourage devs to only release them and forget out-of-the-way publishers like manuals-publisher.

Reply:
Good point, I've updated to include a list and note this.